gui.py

```diff
@@ -155,6 +187,9 @@ def __init__(self, song, jack_client):
         self.readQueueIn.connect(self.readQueue)
         self.current_vol_block = 0
 
+        self.ports_initialised = Event()
```

Comment:
Do you have an alternative suggestion? It seemed the simplest threading tool to solve #7.